src/current/cockroachcloud/create-an-account.md

```diff
@@ -39,17 +38,13 @@ We highly recommend enabling multi-factor authentication (MFA) with your SSO pro
 </section>
 
 <section class="filter-content" markdown="1" data-scope="email">
-1. Enter your **Email address**.
-1. Enter your **Name**.
-1. Enter a **Password**.
-1. Enter your **Organization name (optional)**.
-
-    This can be [changed](#change-your-organization-name) later.
-
-1. Select the checkbox to accept the [terms of service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [privacy policy](https://www.cockroachlabs.com/privacy).
+1. On the [registration page](https://cockroachlabs.cloud/signup?referralId=docs_create_account), click **Sign up with email**.
+1. Enter your **Work email** and a unique **Password** for CockroachDB Cloud.
```

Comment:
Is there a reason we need to be this prescriptive?
```suggestion
1. Enter an email address and password for CockroachDB Cloud.
```

Reply:
Confused, because I thought I made the same edit! Thanks.